python/src/worker/response.py

```diff
@@ -61,58 +56,67 @@ def _upload(self, result):
         if was_enabled:
             xray.global_sdk_config.set_sdk_enabled(False)
 
-        client.put_object(Key=key, Bucket=self.s3_bucket, Body=body)
+        client.put_object(Key=ETag, Bucket=self.s3_bucket, Body=body)
+
+        client.put_object_tagging(
+            Key=ETag,
+            Bucket=self.s3_bucket,
+            Tagging={
+                'TagSet': [
+                    {
+                        'Key': 'experimentId',
+                        'Value': self.request['experimentId']
+                    },
+                    {
+                        'Key': 'requestType',
+                        'Value': self.request['body']['name']
+                    },
+
+                    # TODO: this needs to be removed and a proper
+                    # ACL system implemented before merging.
```

Comment:
this comment is inaccurate -- we will be implementing a proper ACL system after merging, most likely in the next sprint

Reply:
fixed